insights/client/constants.py

```diff
@@ -79,8 +79,9 @@ class InsightsConstants(object):
     cached_branch_info = os.path.join(default_conf_dir, '.branch_info')
     pidfile = os.path.join(os.sep, 'var', 'run', 'insights-client.pid')
     insights_tmp_path = os.path.join(os.sep, 'var', 'tmp')
+    cache_dir = os.path.join(os.sep, 'var', 'cache', 'insights-client')
     insights_tmp_prefix = 'insights-client'
```

Comment:
This `insights_tmp_prefix` became an unused variable then, I assume that you will remove it or use it in the `cache_dir` in your mentioned update. :)

Reply:
oh, no, it is used in the code and it can be re-used in the cache_dir constant :smile: